rclcpp/test/rclcpp/test_qos_event.cpp

```diff
@@ -64,6 +69,11 @@ constexpr char TestQosEvent::topic_name[];
  */
 TEST_F(TestQosEvent, test_publisher_constructor)
 {
+  std::string rmw_implementation_str = std::string(rmw_get_implementation_identifier());
+  if (rmw_implementation_str == "rmw_zenoh_cpp") {
+    GTEST_SKIP();
+  }
+
```

Comment:
@ahcorde just curious, can we have this code in `Setup` function cz it seems all tests are skipped with rmw_zenoh?